libs/search-widget/src/core/search-bar.ts

```diff
@@ -62,10 +61,10 @@ export const setupTriggerSearch = (
                       ...options,
                       show,
                       filters,
-                      inTitleOnly: isTitleOnly(query, labelFilters),
+                      inTitleOnly: !query && allFiltersApplyToResource,
```

Comment:
this doesn't make much sense to me: I remember we search in title only when there is no query, but I don't understand why we also check that all filters are applied

Reply:
Maybe I didn't chose a good name. `allFiltersApplyToResource` is true when all current filters are labels of kind `resource`